3ds Max/Max2Babylon/MaxScriptManager.cs

```diff
@@ -76,8 +78,8 @@ public static ExportParameters InitParameters(string outputPath)
             exportParameters.autoSave3dsMaxFile = Loader.Core.RootNode.GetBoolProperty("babylonjs_autosave");
             exportParameters.exportOnlySelected = Loader.Core.RootNode.GetBoolProperty("babylonjs_onlySelected");
             exportParameters.exportTangents = Loader.Core.RootNode.GetBoolProperty("babylonjs_exporttangents");
-            exportParameters.scaleFactor = Loader.Core.RootNode.GetStringProperty("babylonjs_txtScaleFactor", "1");
-            exportParameters.txtQuality = Loader.Core.RootNode.GetStringProperty("babylonjs_txtCompression", "100");
+            exportParameters.scaleFactor = float.Parse(Loader.Core.RootNode.GetStringProperty("babylonjs_txtScaleFactor", "1"));
```

Comment:
Should that be TryParse in case not float values are in to prevent uncaught exceptions ?

(valid for the next line as well)

Reply:
Yes, we should make sure we don't throw exceptions. I'll change that to TryParse